examples/prometheus/prometheus.yaml

```diff
@@ -224,6 +224,21 @@ objects:
           - --web.listen-address=localhost:9090
           image: ${IMAGE_PROMETHEUS}
           imagePullPolicy: IfNotPresent
+          livenessProbe:
+            exec:
+              command:
+              - /bin/sh
+              - -i
```

Comment:
Our convention for this is:

```
command:
- /bin/bash
- -c
args:
- |
  #!/bin/bash
  set -euo pipe fail
  ....
```

Reply:
It appears the exec liveness probe only supports 'command', not 'args'.